src/web/topic/components/Edge/ScoreEdge.tsx

```diff
@@ -137,6 +138,10 @@ export const ScoreEdge = ({ inReactFlow, ...flowEdge }: EdgeProps & Props) => {
       strokeOpacity={0}
       strokeWidth={20}
       onClick={() => setSelected(edge.id)}
+      onContextMenu={(event) => {
+        console.log("Edge context menu:", edge);
+        openContextMenu(event, { edge });
+      }}
```

Comment:
Shouldn't need this `console.log`
```suggestion
      onContextMenu={(event) => openContextMenu(event, { edge })}
```

Reply:
You are right...We don't need this here..sorry

Reply:
Why its showing commit is outdated?

Reply:
GitHub shows a comment as outdated if it's on a line of code that has been changed since the comment was made. I commented on the `console.log` which you've removed, so the comment is now outdated.